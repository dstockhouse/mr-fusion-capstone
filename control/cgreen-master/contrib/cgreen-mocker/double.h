double return_double(double d, int i);
