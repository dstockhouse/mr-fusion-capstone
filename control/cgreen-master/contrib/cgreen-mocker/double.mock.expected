/* -*- c -*-*/
#include "double.h"
#include <cgreen/mocks.h>

double return_double(double d, int i) { 
  return unbox_double(mock(box_double(d), i);
}

